spec/models/spotlight/exhibit_spec.rb

```diff
@@ -299,4 +299,26 @@
   it 'is expected to be versioned' do
     is_expected.to be_versioned
   end
+  describe 'translatable fields' do
+    let(:persisted_exhibit) { FactoryBot.create(:exhibit, title: 'Sample', subtitle: 'SubSample') }
+    before do
+      FactoryBot.create(:translation, locale: 'fr', exhibit: persisted_exhibit, key: "#{persisted_exhibit.slug}.title", value: 'Titre français')
+      FactoryBot.create(:translation, locale: 'fr', exhibit: persisted_exhibit, key: "#{persisted_exhibit.slug}.subtitle", value: 'Sous-titre français')
+    end
+    after do
+      I18n.locale = 'en'
```

Comment:
Another approach here (that doesn't require teardown) would be to do a `allow` / `expect` in the test (e.g. `expect(I18n).to receive(:locale).and_return(:fr)`).

Not a huge deal, but ran into this in my other test and found it a bit cleaner to stub instead of setup/teardown.